### PR TITLE
scripts for remote benchmarking

### PR DESCRIPTION
**Pull request by schasse**

### CODE REVIEW

**Review by Peterschuellermann — CHANGES_REQUESTED** [review]
apt-get will not work for AWS which uses yum. Please add distinction.

File: benchmark.sh [hunk 1]
```diff
@@ -7,14 +9,16 @@ if [[ $UID != 0 ]]; then
...
 
-#if debian?
-#sudo yam update
-#sudo yam upgrade
-#sudo yam install gcc
+printf "Date: " > results.txt
+date >> results.txt
 
-printf "Benchmark for: " > results.txt
+printf "Benchmark for: " >> results.txt
```
Comment (Peterschuellermann) [inline benchmark.sh:21]: ">" was correct so it overwrites the last results file and not appends to ti

File: benchmark.sh [hunk 2]
```diff
@@ -30,8 +34,4 @@ printf "\nmemsweep:\n">> results.txt
 sh memsweep.sh | grep "seconds" >> results.txt
 
 printf "\nHardware:\n" >> results.txt
-sudo lshw -short >> results.txt
-
-
-rm linpack
```
Comment (Peterschuellermann) [inline benchmark.sh:36]: the compiled versions might interfere with rsync and are therefore deleted beforehand

File: benchmark.sh (hunk 1 above)
Comment (schasse) [reply benchmark.sh:21]: see line `19`. there is the overwriting `>`.

File: benchmark.sh (hunk 2 above)
Comment (schasse) [reply benchmark.sh:36]: `remote_benchmark.sh` uses git to upload `results.txt` and the binaries are in `.gitignore`.

File: benchmark.sh [hunk 3]
```diff
@@ -7,14 +9,16 @@ if [[ $UID != 0 ]]; then
     exit 1
 fi
 
-echo "running Benchmark ans saving results to results.txt"
+echo "running Benchmark and saving results to results.txt"
+
+# prepare instance and install deps
+apt-get update
+apt-get install -y build-essential
```
Comment (schasse) [inline benchmark.sh]: on openstack we need to use apt-get.

Comment (schasse) [reply benchmark.sh]: the assignment says we should use 'amazon linux ami'. i don't know which package manager it uses.

**Review by Peterschuellermann — APPROVED** [review]

**Pull request merged**